pkg/manifests/manifests_test.go

```diff
@@ -2814,7 +2814,13 @@ metricsServer:
 			"tls.key": []byte("foo"),
 		},
 	}
-	d, err := f.MetricsServerDeployment(kubeletCABundle, servingCASecret, metricsClientSecret)
+	apiAuthConfigMapData := map[string]string{
+		"requestheader-allowed-names":        "",
+		"requestheader-extra-headers-prefix": "",
+		"requestheader-group-headers":        "",
+		"requestheader-username-headers":     "",
+	}
+	d, err := f.MetricsServerDeployment("foo", kubeletCABundle, servingCASecret, metricsClientSecret, apiAuthConfigMapData)
```

Comment:
if MetricsServerDeployment apiAuthSecretName param gor removed, it should be also removed on the tests.

Reply:
no that was a need a needed parameter, I fixed the code in MetricsServerDeployment() :)

Reply:
oh, I didn't get to see the fix there, thanks!